sdk/cosmosdb/cosmos/src/retry/resourceThrottleRetryPolicy.ts

```diff
@@ -51,6 +55,7 @@ export class ResourceThrottleRetryPolicy {
 
         if (this.cummulativeWaitTimeinMs < this.timeoutInMs) {
           this.cummulativeWaitTimeinMs += this.retryAfterInMs;
+          diagnosticNode.addData({ successfulRetryPolicy: "resourceThrottle" });
```

Comment:
@v1k1 should we add this  diagnosticNode.addData({ successfulRetryPolicy: "resourceThrottle" }); statement after this if statement?